test/Microsoft.AspNetCore.DataProtection.Extensions.Test/DataProtectionProviderTests.cs

```diff
@@ -38,6 +38,46 @@ public void System_UsesProvidedDirectory()
         [ConditionalFact]
         [ConditionalRunTestOnlyIfLocalAppDataAvailable]
         [ConditionalRunTestOnlyOnWindows]
+        public void System_UsesDefaultDirectory()
+        {
+            var keysPath = Path.Combine(Environment.ExpandEnvironmentVariables("%LOCALAPPDATA%"), "ASP.NET", "DataProtection-Keys");
```

Comment:
This was the best way to test this :smile: . Also it preserves the existing files in that folder. So its not that bad.